flashinfer_bench/serve/low_level/server.py

```
@@ -0,0 +1,285 @@
+                            binary_parts=success_response.binary_parts,
+                        )
+            except TimeoutError as error:
+                server_response = ServerExecuteResponse(
+                    kind=ServerExecuteResponseKind.TIMEOUT,
+                    payload=ErrorResult(error="timeout", timeout_seconds=error.timeout_seconds),
+                )
+            except asyncio.CancelledError:
+                if not pending_task.future.done():
+                    pending_task.future.cancel()
+                raise
+            except Exception as error:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)